client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -138,7 +135,7 @@ const NewsletterSetup: Step = ( { navigation } ) => {
 						onChangeComplete={ ( value ) => setAccentColor( value.hex ) }
 					/>
 				</Popover>
-				<FormFieldset disabled={ isLoading }>
+				<FormFieldset disabled={ false }>
```

Comment:
We could have a boolean `isSubmitting` that becomes true and disables the input `onSubmit`!

Reply:
I think we can just remove it completely once we are moving the submit part to another place.